### PR TITLE
Fix/landing-page-style

### PR DESCRIPTION
**Pull request by ashutoshamazix**
## Description

- Fixed CSS Style for Safari
- Added scroll restoration for landing pages
- Mock Test for Slick Slider/matchMedia

## How Has This Been Tested?

- Unit test and manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by IAmKio — APPROVED** [review]
LGTM but waiting for @RanaBug's approval

**Review by RanaBug — APPROVED** [review]

**Pull request merged**
